game/src/ecs/systems/ECS_System.java

```diff
@@ -1,11 +1,13 @@
 package ecs.systems;
 
+import java.util.logging.Logger;
 import starter.Game;
 
 /** Marks a Class as a System in the ECS */
 public abstract class ECS_System {
 
     protected boolean run;
+    public Logger systemLogger = Logger.getLogger(this.getClass().getSimpleName());
```

Comment:
s.o. für eins entscheiden

Reply:
s.o. fixed in 52f4671d701138239da1e0e6cc93a3e66b8bd1a8